### PR TITLE
Handle Negative Zero Decimal

### PR DESCRIPTION
**Pull request by justing-bq**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: ion/decimal.go [hunk 1]
```diff
@@ -38,21 +38,23 @@ func (e *ParseError) Error() string {
 
 // Decimal is an arbitrary-precision decimal value.
 type Decimal struct {
-	n     *big.Int
-	scale int32
+	n         *big.Int
+	scale     int32
+	isNegZero bool
```
Comment (Armanbqt) [inline ion/decimal.go:43]: Do you think we can use ```isNegZero``` in ```cmp()``` function?

Comment (justing-bq) [reply ion/decimal.go:43]: Addressed.

File: ion/integration_test.go [hunk 2]
```diff
@@ -124,11 +124,7 @@ var equivsSkipList = []string{
 	"systemSymbols.ion",
 }
 
-var nonEquivsSkipList = []string{
-	"decimals.ion",
-	"floats.ion",
-	"floatsVsDecimals.ion",
-}
+var nonEquivsSkipList = []string{}
```
Comment (Armanbqt) [inline ion/integration_test.go:127]: ❤

Comment (justing-bq) [reply ion/integration_test.go:127]: Feels good to see an empty skiplist.

**Review by therapon — APPROVED** [review]

**Review by zslayton — APPROVED** [review]

**Pull request merged**